solidity/dashboard/src/pages/tbtc-migration/HowItWorksPage.jsx

```diff
@@ -0,0 +1,166 @@
+import React from "react"
+import List from "../../components/List"
+import * as Icons from "../../components/Icons"
+import NavLink from "../../components/NavLink"
+import Divider from "../../components/Divider"
+
+const tbtcPools = [
+  {
+    id: "v2",
+    icon: Icons.SaddleWhite,
+    label: "tBTC v2 Pool on Saddle",
+    btnText: "go to pool ↗",
+    link: "https://saddle.exchange/#/pools/btc/deposit",
+    external: true,
+  },
+  {
+    id: "v1",
```

Comment:
"v1" nad "v2" also used here, so I would use the const here also as stated few comments before